Simple-PHP-IPAM/assets/app.css

```
@@ -558,7 +558,8 @@ button:disabled{opacity:.6;cursor:not-allowed}
 .badge-network{background:color-mix(in srgb,var(--muted) 15%,var(--badge-bg) 85%);color:var(--muted);font-size:.7rem;padding:2px 6px}
 .badge-broadcast{background:color-mix(in srgb,var(--warn) 15%,var(--badge-bg) 85%);color:var(--warn);font-size:.7rem;padding:2px 6px}
 .badge-gateway{background:color-mix(in srgb,var(--link) 15%,var(--badge-bg) 85%);color:var(--link);font-size:.7rem;padding:2px 6px}
-.audit-details{display:inline-block;max-width:300px;overflow:hidden;text-overflow:ellipsis;white-space:nowrap;vertical-align:bottom}
+.audit-details{display:inline-block;max-width:300px;overflow:hidden;text-overflow:ellipsis;white-space:nowrap;vertical-align:bottom;cursor:pointer}
+.audit-details.audit-details--expanded{max-width:none;white-space:normal;word-break:break-word;cursor:pointer}
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)